src/Auth.ts

```diff
@@ -453,7 +453,9 @@ export class Auth {
       await logger.logToStderr('');
     }
 
-    await logger.logToStderr(`🌶️  ${response.message}`);
+    if (response.message) {
+      await logger.logToStderr(`🌶️  ${response.message}`);
```

Comment:
A quick question, can we drop the two spaces between the emoji and the message? My OCD can't handle this 😄

Reply:
We use two spaces, because the chili emoji takes 1,5 char so with a single space in some cases it feels like there's no space between the emoji and the text that follows it.